patches/aspnetcore-tooling/0005-remove-unneeded-feeds.patch

```diff
@@ -1,30 +1,29 @@
-From bb94c04c907944ea41ba0a4786e5a4803a7a5d7c Mon Sep 17 00:00:00 2001
+From 1d4321a450868d66ecb82be2da9ffeda34b2ad99 Mon Sep 17 00:00:00 2001
 From: Aditya Aggarwal <aditya.aggarwal@microsoft.com>
 Date: Mon, 16 Mar 2020 04:48:31 +0000
 Subject: [PATCH 5/5] remove unneeded feeds
 
 ---
- NuGet.config | 7 -------
- 1 file changed, 7 deletions(-)
+ NuGet.config | 6 ------
+ 1 file changed, 6 deletions(-)
 
 diff --git a/NuGet.config b/NuGet.config
-index c17697d78..f140cdc4f 100644
+index 3df8fa2d0..f140cdc4f 100644
 --- a/NuGet.config
 +++ b/NuGet.config
-@@ -2,13 +2,6 @@
+@@ -2,12 +2,6 @@
  <configuration>
    <packageSources>
      <clear />
 -    <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
--    <add key="darc-pub-dotnet-extensions-7d9baaf" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-extensions-7d9baaf6/nuget/v3/index.json" />
--    <add key="darc-pub-dotnet-core-setup-4a9f85e" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-core-setup-4a9f85e9/nuget/v3/index.json" />
--    <add key="darc-pub-dotnet-corefx-e946ceb-1" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-corefx-e946cebe-1/nuget/v3/index.json" />
 -    <add key="darc-pub-dotnet-corefx-8a3ffed" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-corefx-8a3ffed5/nuget/v3/index.json" />
--    <add key="darc-pub-dotnet-corefx-0f7f38c" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-corefx-0f7f38c4/nuget/v3/index.json" />
+-    <add key="darc-int-dotnet-extensions-3b1f2b7" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-extensions-3b1f2b7c/nuget/v3/index.json" />
+-    <add key="darc-int-dotnet-core-setup-0c2e69c" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-core-setup-0c2e69ca/nuget/v3/index.json" />
+-    <add key="darc-int-dotnet-corefx-059a4a1" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-corefx-059a4a19/nuget/v3/index.json" />
```

Comment:
I think we should stop using patches for this, filed https://github.com/dotnet/source-build/issues/1594.